.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -15,6 +15,10 @@ steps:
       filePath: yarn-ci.sh
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31205&commitId=b7b1337deafeb4b68239900847bd26b996432438&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Personas.default.chromium.png|5| Changed|
|ActivityItem.CompactPersonas.default.chromium.png|3| Changed|
|ActivityItem.Personas - RTL.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Checkbox</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Custom render Checkbox.default.chromium.png|2| Changed|

</details>


  <details><summary><b>ChoiceGroup</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.With default size images.selected.chromium.png|184| Changed|
|ChoiceGroup.With default size images.default.chromium.png|168| Changed|
|ChoiceGroup.With default size images.hover unselected.chromium.png|183| Changed|
|ChoiceGroup.With default size images.hover selected.chromium.png|184| Changed|

</details>


  <details><summary><b>Coachmark</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|2| Changed|
|Coachmark.Rendering Coachmark attached to a rectangle.default.chromium.png|1| Changed|

</details>


  <details><summary><b>ColorPicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ColorPicker.Blue.default.chromium.png|2| Changed|

</details>


  <details><summary><b>ComboBox</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Error.click.chromium.png|111| Changed|
|ComboBox.Styled.click.chromium.png|111| Changed|
|ComboBox.Root.click.chromium.png|111| Changed|
|ComboBox.Root - RTL.click.chromium.png|111| Changed|

</details>


  <details><summary><b>Facepile</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Facepile.Extra extra small.default.chromium.png|5| Changed|
|Facepile.Root.default.chromium.png|5| Changed|
|Facepile.Add face.default.chromium.png|7| Changed|
|Facepile.Overflow.default.chromium.png|2| Changed|
|Facepile.Root - RTL.default.chromium.png|5| Changed|
|Facepile.Custom button styles.default.chromium.png|2| Changed|

</details>


  <details><summary><b>Fonts</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Fonts.Chinese (Simplified).default.chromium.png|11191| Changed|
|Fonts.Hebrew.default.chromium.png|11347| Changed|
|Fonts.Armenian.default.chromium.png|10615| Changed|
|Fonts.Georgian.default.chromium.png|11193| Changed|
|Fonts.Hindi.default.chromium.png|14872| Changed|
|Fonts.Cyrillic.default.chromium.png|11191| Changed|
|Fonts.Korean.default.chromium.png|14872| Changed|
|Fonts.Thai.default.chromium.png|8397| Changed|
|Fonts.Chinese (Traditional).default.chromium.png|11191| Changed|
|Fonts.Greek.default.chromium.png|11191| Changed|
|Fonts.Arabic.default.chromium.png|11347| Changed|
|Fonts.Japanese.default.chromium.png|23677| Changed|
|Fonts.East European.default.chromium.png|11183| Changed|
|Fonts.Vietnamese.default.chromium.png|11159| Changed|
|Fonts.West European.default.chromium.png|11191| Changed|

</details>


  <details><summary><b>PeoplePicker</b> 17 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.List selected - RTL.default.chromium.png|1| Changed|
|PeoplePicker.Compact selected - RTL.default.chromium.png|1| Changed|
|PeoplePicker.Normal Overflow selected - RTL.default.chromium.png|2| Changed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|1| Changed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|2| Changed|
|PeoplePicker.Compact selected.suggestions.chromium.png|9| Changed|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|7| Changed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|161| Changed|
|PeoplePicker.Compact.suggestions.chromium.png|8| Changed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|8| Changed|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|10| Changed|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|9| Changed|
|PeoplePicker.List.suggestions.chromium.png|6| Changed|
|PeoplePicker.Normal.suggestions.chromium.png|6| Changed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|8| Changed|
|PeoplePicker.List selected.suggestions.chromium.png|6| Changed|
|PeoplePicker.Normal selected.suggestions.chromium.png|7| Changed|

</details>


  <details><summary><b>Persona</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.size100 (extraLarge) - RTL.default.chromium.png|3| Changed|
|Persona.size100 (extraLarge).default.chromium.png|3| Changed|
|Persona.size28 (extraSmall).default.chromium.png|1| Changed|
|Persona.size24 (extraExtraSmall).default.chromium.png|1| Changed|
|Persona.size72 (large).default.chromium.png|3| Changed|

</details>


  <details><summary><b>Slider</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.EqualMinMax.default.chromium.png|1| Changed|
|Slider.EqualMinMax.hover.chromium.png|2| Changed|
|Slider.Max not multiple of step.hover.chromium.png|2| Changed|
|Slider.Root - RTL.hover.chromium.png|2| Changed|
|Slider.Root.default.chromium.png|1| Changed|
|Slider.Root.hover.chromium.png|2| Changed|
|Slider.Step less than 1.hover.chromium.png|2| Changed|
|Slider.Step less than 1.default.chromium.png|1| Changed|
|Slider.Vertical.hover.chromium.png|2| Changed|
|Slider.Root - RTL.default.chromium.png|1| Changed|
|Slider.Vertical.default.chromium.png|1| Changed|
|Slider.Max not multiple of step.default.chromium.png|1| Changed|

</details>


  <details><summary><b>SwatchColorPicker</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchColorPicker.Circle - RTL.Focused.chromium.png|7| Changed|
|SwatchColorPicker.Circle - RTL.Selected and Focused.chromium.png|9| Changed|
|SwatchColorPicker.Circle - RTL.Hovered.chromium.png|7| Changed|
|SwatchColorPicker.Circle over 24px size.Focused.chromium.png|12| Changed|
|SwatchColorPicker.Circle - RTL.Mousedown.chromium.png|7| Changed|
|SwatchColorPicker.Circle - RTL.default.chromium.png|7| Changed|
|SwatchColorPicker.Circle over 24px size.Selected and Focused.chromium.png|16| Changed|
|SwatchColorPicker.Circle.Focused.chromium.png|7| Changed|
|SwatchColorPicker.Circle over 24px size.Mousedown.chromium.png|13| Changed|
|SwatchColorPicker.Circle.Mousedown.chromium.png|7| Changed|
|SwatchColorPicker.Circle.Hovered.chromium.png|7| Changed|
|SwatchColorPicker.Circle over 24px size.default.chromium.png|14| Changed|
|SwatchColorPicker.Circle.Selected and Focused.chromium.png|9| Changed|
|SwatchColorPicker.Circle over 24px size.Selected and Hovered.chromium.png|16| Changed|
|SwatchColorPicker.Multiple rows.Mousedown.chromium.png|14| Changed|
|SwatchColorPicker.Multiple rows.Selected and Focused.chromium.png|16| Changed|
|SwatchColorPicker.Multiple rows.Hovered.chromium.png|14| Changed|
|SwatchColorPicker.Multiple rows.Focused.chromium.png|14| Changed|
|SwatchColorPicker.Multiple rows.default.chromium.png|14| Changed|
|SwatchColorPicker.Circle.Selected and Hovered.chromium.png|9| Changed|
|SwatchColorPicker.Circle.default.chromium.png|7| Changed|
|SwatchColorPicker.Multiple rows.Selected and Hovered.chromium.png|16| Changed|
|SwatchColorPicker.Circle - RTL.Selected and Hovered.chromium.png|9| Changed|
|SwatchColorPicker.Circle over 24px size.Hovered.chromium.png|13| Changed|

</details>


  <details><summary><b>Tile</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tile.Document tile with icon.default.chromium.png|4| Changed|

</details>


  <details><summary><b>Toggle</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toggle.Checked - RTL.default.chromium.png|2| Changed|
|Toggle.Checked.default.chromium.png|2| Changed|
|Toggle.With inline label and without onText and offText.default.chromium.png|2| Changed|
|Toggle.With inline label.default.chromium.png|2| Changed|
|Toggle.With inline label (JSX Element).default.chromium.png|2| Changed|
|Toggle.With inline label (JSX Element) and without onText and offText.default.chromium.png|2| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic.hover.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-GaugeChart</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-GaugeChart.Basic.default.chromium.png|1| Changed|
|react-charting-GaugeChart.Variant.default.chromium.png|1| Changed|
|react-charting-GaugeChart.Variant - RTL.default.chromium.png|1| Changed|
|react-charting-GaugeChart.Basic - RTL.default.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic.hover.chromium.png|3| Changed|
|react-charting-LineChart.Events - RTL.default.chromium.png|1| Changed|
|react-charting-LineChart.Gaps - RTL.default.chromium.png|3| Changed|
|react-charting-LineChart.Basic - RTL.default.chromium.png|3| Changed|
|react-charting-LineChart.Basic.default.chromium.png|3| Changed|
|react-charting-LineChart.Multiple - RTL.default.chromium.png|61| Changed|
|react-charting-LineChart.Events.default.chromium.png|2| Changed|
|react-charting-LineChart.Gaps.default.chromium.png|4| Changed|
|react-charting-LineChart.Multiple.default.chromium.png|60| Changed|
|react-charting-LineChart.Multiple - Dark Mode.default.chromium.png|45| Changed|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Basic - RTL.default.chromium.png|22| Changed|
|react-charting-SankeyChart.Basic - Dark Mode.default.chromium.png|8| Changed|
|react-charting-SankeyChart.Basic.default.chromium.png|23| Changed|

</details>


  <details><summary><b>react-charting-SparkLineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|1| Changed|
|react-charting-SparkLineChart.Basic.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>